### DISCUSSION TITLE
使用caddy反代web渲染异常

### DISCUSSION THREAD

**Discussion by LoadingZhang** [qd-today/qd#218] [Q&A]
加了反代成这样了，无法正常登陆![image](https://user-images.githubusercontent.com/10226902/152802524-fd0b4889-9623-42e0-8204-6c87c000a41d.png)
控制台报错
![image](https://user-images.githubusercontent.com/10226902/152804776-aef37dfe-c0d8-4909-b025-d18068d33d3f.png)
跳过caddy直接访问签到服务正常，这是为什么？

**Comment by a76yyyy** [comment]
> 

看一下你的规则里是不是只反代了首页而没有反代二级页面
